src/js/Controllers/RpcFactory.js

```diff
@@ -1,6 +1,16 @@
 import {capabilities, getDisplayCapability} from './DisplayCapabilities.js'
 var rpcFactory_msgId = 5012
 class RpcFactory {
+    static BCSetAppProperties(properties) {
```

Comment:
Is the BC prefix necessary?

Reply:
removed in https://github.com/smartdevicelink/generic_hmi/pull/240/commits/9420e8cd33100df3cba2420ebea12091814adcac